pkg/controller/data_download_controller.go

```diff
@@ -86,12 +87,19 @@ func NewDataDownloadReconciler(
 	preparingTimeout time.Duration,
 	logger logrus.FieldLogger,
 	metrics *metrics.ServerMetrics,
+	dataMovePriorityClass string,
 ) *DataDownloadReconciler {
+	log := logger.WithField("controller", "DataDownload")
```

Comment:
It doesn't look like necessary as we have already printed the same log in Velero server startup or node-agent server startup.